test/ibmq/test_ibmq_job.py

```diff
@@ -267,7 +267,7 @@ def test_retrieve_multiple_job_statuses(self):
 
         job_to_cancel = submit_and_cancel(backend=self.sim_backend)
         job_to_fail = submit_job_bad_shots(backend=self.sim_backend)
-        job_to_fail.wait_for_final_state()
+        job_to_fail.cancel()
```

Comment:
`job_to_fail` is meant to create a failed job, as this test is to retrieve jobs in different statuses

Reply:
Updated! Added a [status check](https://github.com/Qiskit-Partners/qiskit-ibm/pull/9/files#diff-313b7cac7866bc872344a595d0f613f12c5dd1bd6ade62031c8f3e5f7e4330d0R274-R275) for good measure. 😅